.github/workflows/scripts/check-docs.sh

```diff
@@ -31,15 +31,23 @@ if [ -z "$package_files" ]; then
   fatal "Package.swift not found. Please ensure you are running this script from the root of a Swift package."
 fi
 
-for package_file in $package_files; do
-  log "Editing $package_file..."
-  cat <<EOF >> "$package_file"
+# yq 3.1.0-3 doesn't have filter, otherwise we could replace the grep call with "filter(.identity == "swift-docc-plugin") | keys | .[]"
+hasDoccPlugin=$(swift package dump-package | yq -r '.dependencies[].sourceControl' | grep -e "\"identity\": \"swift-docc-plugin\"" || true )
```

Comment:
`swift package dump-package` is JSON, so should this be `jq` instead of `yq`?

Also: Trailing space after `true`)

Reply:
We already use `yq` later on in the script, so I figured no need to an extra dependency, since `yq` supports json (as well as yaml)